modules/ixBidAdapter.js

```diff
@@ -2056,4 +2057,19 @@ function getFormatCount(imp) {
   return formatCount;
 }
 
+/**
+ * Adds device.w / device.h info
+ * @param {object} r
+ * @returns object
+ */
+export function addDeviceInfo(r) {
```

Comment:
why not do this in the global ortb2 object for all bidders picking up device?

Reply:
Thats just going to be a longer process, as I've never touched pbjs core before. Can we merge this in for now and I can work on the core change